src/components/Icon/Arrow.tsx

```diff
@@ -9,19 +9,23 @@ export const rotationDegree = {
   [IconDirection.Right]: "270",
 };
 
-const SvgUp = (props: IconDirectionProps & IconProps) => (
+export type ArrowIconProps = IconDirectionProps & IconProps;
+
+const SvgUp = (props: ArrowIconProps): React.ReactElement => (
```

Comment:
The name of this svg should probably be `SvgArrow`

Reply:
Fixed by 5e05e58